server/cache/cache.go

```diff
@@ -1,26 +1,6 @@
 package cache
 
-import (
-	"context"
-	"time"
-
-	"k8s.io/client-go/informers"
-	"k8s.io/client-go/kubernetes"
-	v1 "k8s.io/client-go/listers/core/v1"
-)
-
-type ResourceCache struct {
-	v1.ServiceAccountLister
-	v1.SecretLister
-}
-
-func NewResourceCache(client kubernetes.Interface, ctx context.Context, namespace string) *ResourceCache {
-	informerFactory := informers.NewSharedInformerFactoryWithOptions(client, time.Minute*20, informers.WithNamespace(namespace))
-	cache := &ResourceCache{
-		ServiceAccountLister: informerFactory.Core().V1().ServiceAccounts().Lister(),
-		SecretLister:         informerFactory.Core().V1().Secrets().Lister(),
-	}
-	informerFactory.Start(ctx.Done())
-	informerFactory.WaitForCacheSync(ctx.Done())
-	return cache
+type Cache interface {
```

Comment:
If a package has a single key interface, I think it should be named `Interface`.

Reply:
you mean `type Interface interface {`
in a file called `Interface.go` ?

Reply:
done